app/src/main/kotlin/de/digitalService/useID/ui/UseIDApp.kt

```diff
@@ -143,15 +129,19 @@ private fun PreviewNfc() {
 @Preview(name = "Large", showSystemUi = true, device = Devices.PIXEL_4_XL)
 @Composable
 private fun PreviewNoNfc() {
-    UseIDApp(appCoordinator = PreviewAppCoordinator(rememberUpdatedState(newValue = NfcAvailability.NoNfc), false), PreviewAppNavigator(), PreviewTrackerManager())
+    UseIDApp(
+        NfcAvailability.Available,
```

Comment:
Shouldn't this be `.NoNfc` ?

Reply:
No, in the Preview we want to simulate any NFC related interaction. The app thinks NFC is available (and therefore doesn't show the NoNFC screen).